vcsclient/bitbucketserver.go

```diff
@@ -598,3 +603,7 @@ func (client *BitbucketServerClient) mapBitbucketServerCommitToCommitInfo(commit
 		ParentHashes:  parents,
 	}
 }
+
+func (client *BitbucketServerClient) UploadCodeScanning(ctx context.Context, owner string, repository string, branch string, scan string) (string, error) {
```

Comment:
Please make sure to do that in all other implementations:
```suggestion
func (client *BitbucketServerClient) UploadCodeScanning(ctx context.Context, owner string, repository string, branch string, scanResults string) (string, error) {
```

Reply:
Done